faq.html

```diff
@@ -73,12 +73,15 @@ <h3 id="aboutRDS" class="question">How do I contribute to RDS?</h3>
 
               <!-- prettier-ignore -->
               <ul class="faq__list">
-                  <li><a href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank" rel="noopener noreferrer">Welcome</a> : HTML, CSS, Javascript</li>
-                  <li><a href="https://github.com/Real-Dev-Squad/website-members" target="_blank" rel="noopener noreferrer">Members</a> : React</li>
-                  <li><a href="https://github.com/Real-Dev-Squad/rds-events-website" target="_blank" rel="noopener noreferrer">Events</a>: React</li>
-                  <li><a href="https://github.com/Real-Dev-Squad/website-crypto" target="_blank" rel="noopener noreferrer">Crypto</a> : Next.js</li>
-                  <li><a href="https://github.com/Real-Dev-Squad/website-emberjs" target="_blank" rel="noopener noreferrer">Main App</a> : Ember</li>
-                  <li><a href="https://github.com/Real-Dev-Squad/website-backend" target="_blank" rel="noopener noreferrer">Backend</a> : Node.js</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-welcome" target="_blank" rel="noopener noreferrer">Welcome</a> : HTML, CSS, Javascript</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-www" target="_blank" rel="noopener noreferrer">Main Site</a> : HTML, CSS, Javascript</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-members" target="_blank" rel="noopener noreferrer">Members</a> : React, Next.js</li>
+                <!-- <li><a href="https://github.com/Real-Dev-Squad/rds-events-website" target="_blank" rel="noopener noreferrer">Events</a>: React</li> -->
+                <li><a href="https://github.com/Real-Dev-Squad/website-crypto" target="_blank" rel="noopener noreferrer">Crypto</a> : Next.js</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-status" target="_blank" rel="noopener noreferrer">Status</a> : Next.js, TypeScript</li>
+                <!-- <li><a href="https://github.com/Real-Dev-Squad/website-emberjs" target="_blank" rel="noopener noreferrer">Main App</a> : Ember</li> -->
+                <li><a href="https://github.com/Real-Dev-Squad/website-my" target="_blank" rel="noopener noreferrer">My Site</a> : Ember</li>
+                <li><a href="https://github.com/Real-Dev-Squad/website-backend" target="_blank" rel="noopener noreferrer">Backend</a> : Node.js</li>
```

Comment:
I like how you have maintained the order of the projects... From simple HTML, CSS to frontend frameworks and then backend 😁